dashboard/app/graphs.go

```diff
@@ -239,6 +241,61 @@ func handleHeatmap(c context.Context, w http.ResponseWriter, r *http.Request, f
 	})
 }
 
+func githubTorvaldsLinuxURI(filePath string, rc covermerger.RepoCommit) string {
+	source := rc.Commit
+	if rc.Commit == "latest" {
```

Comment:
How is it possible to use this?
When I do "commit=latest" I get: "input validation failed: commit: not a hash: len is not 40"

Reply:
Thanks, I removed this check.
"latest" was a console version feature. It is not applicable here.

Reply:
ptal